app/client/src/constants/AppConstants.ts

```diff
@@ -44,6 +46,7 @@ export const MOBILE_MAX_WIDTH = 767;
 export const TABLET_MIN_WIDTH = 768;
 export const TABLET_MAX_WIDTH = 991;
 export const DESKTOP_MIN_WIDTH = 992;
+export const DESIGN_BASE_WIDTH = 1440;
```

Comment:
Design base width feels ambiguous. Can we call it App or Editor instead?
```suggestion
export const EDITOR_BASE_WIDTH = 1440;
```

Reply:
Or even desktop